core-bundle/src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -81,9 +83,9 @@ public function getResponse($blnCheckRequest=false, $blnForceCacheHeaders=false)
 	 *
 	 * @throws UnusedArgumentsException If there are unused route parameters
 	 *
-	 * @internal Do not call this method in your code. It will be made private in Contao 5.0.
+	 * @internal
```

Comment:
This can be removed now ☺️

Reply:
Of course. 🤦‍♂️ Removed in 9091327c48588f977cdf089fd94a7c3ffaefd574.